e2e_playwright/st_builtin_chart_test.py

```diff
@@ -43,8 +43,8 @@ def test_themed_chart_rendering(
 
     # Only test a single selected chart per built-in chart type:
     assert_snapshot(chart_elements.nth(1), name=f"st_builtin_chart-area_chart_themed")
-    assert_snapshot(chart_elements.nth(11), name=f"st_builtin_chart-bar_chart_themed")
-    assert_snapshot(chart_elements.nth(21), name=f"st_builtin_chart-line_chart_themed")
+    assert_snapshot(chart_elements.nth(12), name=f"st_builtin_chart-bar_chart_themed")
+    assert_snapshot(chart_elements.nth(23), name=f"st_builtin_chart-line_chart_themed")
```

Comment:
nit: the `f` can be removed here since they don't need to be functional string. The ruff change fixes this for the other instances in this file as well.